server/services/voip/lib/Helper.ts

```diff
@@ -0,0 +1,33 @@
+import { ServerType, IVoipCallServerConfig, IVoipManagementServerConfig } from '../../../../definition/IVoipServerConfig';
+import { settings } from '../../../../app/settings/server';
+
+export function getServerConfigDataFromSettings(type: ServerType): IVoipCallServerConfig | IVoipManagementServerConfig {
+	switch (type) {
+		case ServerType.CALL_SERVER: {
+			const serverCofig: IVoipCallServerConfig = {
+				type: ServerType.CALL_SERVER,
+				host: settings.get<string>('VoIP_Server_Host'),
+				name: settings.get<string>('VoIP_Server_Name'),
+				configData: {
+					websocketPort: settings.get<number>('VoIP_Server_Websocket_Port'),
```

Comment:
```suggestion
					websocketPort: Number(settings.get<number>('VoIP_Server_Websocket_Port')),
```

Reply:
Fixed. But I have a question. If we are already reading it as a number, why do we want to convert it to number again by using Number()?

Reply:
Oh, that's because we already have those values as strings on our db, so i didn't want to disrupt the work. Before merging into develop, this can be removed as clients will have those values already as numbers